packages/standards/biome-react.jsonc

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "./node_modules/@biomejs/biome/configuration_schema.json",
+  "linter": {
+    "domains": {
+      "react": "all",
+    },
+    "rules": {
+      "performance": {
+        // Namespace imports (`import * as foo from 'foo'`) can severely impact performance and
+        // tree-shaking capabilities in various bundlers. We had a prior rule within our
+        // `@readme/eslint-plugin` package, `readme/no-wildcard-imports`, that enforced this. This
+        // is the Biome equivalent.
+        //
+        // https://biomejs.dev/linter/rules/no-namespace-import/
+        "noNamespaceImport": "error",
+      },
+      "nursery": {
+        "noNoninteractiveElementInteractions": "error",
```

Comment:
This is a rule that we had enabled in `@readme/eslint-config/react` via the recommended config from [eslint-plugin-jsx-a11y](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y), [jsx-a11y/no-noninteractive-element-interactions](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/no-noninteractive-element-interactions.md), has a Biome variant but unlike `eslint-plugin-jsx-a11y` it is neither part of its recommended config or on by default in the [`react` domain.](https://biomejs.dev/linter/domains/#react)